db/seeds.sql

```diff
@@ -21,7 +21,8 @@ SET SQL_MODE="NO_AUTO_VALUE_ON_ZERO";
 --
 INSERT INTO `npl_users` (`id`, `username`, `password`, `salt`, `mail`, `active`, `mail_verified`, `register_datetime`, `written_datetime`) VALUES
 (1, 'admin', SHA1(CONCAT('banane', SHA1('banane'))), SHA1('banane'), 'admin@noproblan.ch', 1, 1, '2012-01-01 00:00:00', '2012-01-01 00:00:00'),
-(2, 'guest', SHA1(CONCAT('banane', SHA1('banane'))), SHA1('banane'), 'noreply@noproblan.ch', 1, 1, '2012-01-01 00:00:00', '2012-01-01 00:00:00');
+(2, 'guest', SHA1(CONCAT('banane', SHA1('banane'))), SHA1('banane'), 'noreply@noproblan.ch', 1, 1, '2012-01-01 00:00:00', '2012-01-01 00:00:00'),
+(3, 'sadi', SHA1(CONCAT('banane', SHA1('banane'))), SHA1('banane'), 'sadi@noproblan.ch', 1, 1, '2017-01-01 00:00:00', '2017-01-01 00:00:00');
```

Comment:
The user needs to be added and he needs to get the correct group as well (group id should be 3 for a gamer)